test/release/local_release_page_list.txt

```diff
@@ -7,3 +7,4 @@ scroll_zoom_blocker.html
 threejs-antenna.html
 video.html
 custom-source.html
+heatmap-layer.html
```

Comment:
:+1: 

Concerning the other features to cover, are you planning on adding the other test pages in separate PRs or update this one?

Reply:
Separate PRs for each was what I was planning.

Reply:
Ok sounds good!